packages/serverpod/lib/src/server/websocket_request_handlers/method_websocket_request_handler.dart

```diff
@@ -240,6 +241,23 @@ class MethodWebsocketRequestHandler {
       };
     }
 
+    var authenticationRequired = endpointConnector.endpoint.requireLogin ||
+        endpointConnector.endpoint.requiredScopes.isNotEmpty;
+    _RequiredAuthentication? requiredAuthentication;
+    if (authenticationRequired) {
```

Comment:
Maybe it's just me, but these names are so similar (just the inverse of eachother) that I got a bit confused. Maybe adding an `is` to accentuate that one is a boolean and the other an object could make it a bit easier on the eye 👁️ 

```suggestion
    _RequiredAuthentication? requiredAuthentication;
    
    var authenticationIsRequired = endpointConnector.endpoint.requireLogin ||
        endpointConnector.endpoint.requiredScopes.isNotEmpty;
    if (authenticationIsRequired) {
```